guides/earn-grc.htm

```diff
@@ -52,89 +52,149 @@ <h5>Step 4</h5>
             <!--Step 1-->
             <div class='row'>
                 <div class='col-12 col-sm-12 text-center'>
-                    <h4 class="text-bold text-center">
+                    <h2 class="text-bold text-center">
                         Steps to Earn GRC via DPOR
-                    </h4>
+                    </h2>
                 </div>
             </div>
             <!--/Step 1-->
             <!--Contibute content-->
             <div class="row">
                 <div class="col-12 col-sm-12">
-                    <ul>
+                    <h5 class="text-center"> 1) Configure the Wallet </h5>
+                        <div class="text-center mb-4"><i><small>If you don't use the GUI, go to <a href="#non-gui">the non-gui steps</a>. Ignore this if you don't know what this means</small></i></div>
+                        a) In the wallet, click on the beacon button and then select Solo ("Beacon" may display as "Action Needed")
+                        <img class="img-fluid mx-auto d-block" src="/assets/img/guide/beacon-config-start.png" alt="image with three options and the solo button selected">
+                        <figcaption class="text-center mb-4">
+                            If your wallet doesn't look like the image above, press the "Start Over" button
+                        </figcaption>
+                        b) Press next and enter the email you used on your BOINC projects
+                        <img class="img-fluid mx-auto d-block" src="/assets/img/guide/beacon-email.png" alt="image showing a textbox asking for your BOINC email">
+                        <figcaption class="text-center mb-4">  
+                            Enter your email for your BOINC projects, and don't enter example@example.com
+                        </figcaption> 
+                        c) Click next and make sure you see a check mark (✔️) next to the field labeled CPID. If you do not, go to the <a href="#troubleshooting">troubleshooting section</a> 
+                        <img class="img-fluid mx-auto d-block" src="/assets/img/guide/beacon-overview.png" alt="image showing an overview of information in the wallet about your BOINC account">
+                        <figcaption class="text-center mb-4">
+                            Note that your CPID and email will be different than in the image above
+                        </figcaption>
+                        d) Click next and then press the advertise beacon button
+                        <img class="img-fluid mx-auto d-block mb-4" src="/assets/img/guide/send-beacon.png" alt="image showing a button that has send beacon on it">
+                        e) You should now see the screen below saying you have a pending beacon. If you see any errors go to the <a href="#troubleshooting">troubleshooting section</a> 

```

Comment:
Not sure if it's just me, But on my local tests this `troubleshooting` link does not work.

Reply:
Oops on that one. I accidentally put "mb-3" in the anchor's id in ead17ca815e1d8dccaae2c6c58dc81a1bfd7d3ee. Fixed that now